src/app/shell/admin-tools/data/directions-wrapper/directions-institution-hierarchies-list/directions-institution-hierarchies-list.component.scss

```diff
@@ -0,0 +1 @@
+@import "src/app/shared/styles/tables.scss";
```

Comment:
please use relative import if it possible

Reply:
with styles should be as it is